queries/futures/useGetFuturesPositionForMarkets.ts

```diff
@@ -28,24 +28,21 @@ const useGetFuturesPositionForMarkets = (options?: UseQueryOptions<FuturesPositi
 	const isL2 = useIsL2();
 	const synthetixjs = isL2 ? defaultSynthetixjs : l2Synthetixjs;
 
-	const setFuturesPositions = useSetRecoilState(positionsState);
 	const futuresMarkets = useRecoilValue(futuresMarketsState);
-	const selectedFuturesAddress = useRecoilValue(selectedFuturesAddressState);
-
+	const [positions, setPositions] = useRecoilState(positionsState);
+	const { walletAddress, crossMarginAddress, crossMarginAvailable, status } = useRecoilValue(
+		futuresAccountState
+	);
 	const assets = futuresMarkets.map(({ asset }) => asset);
 
-	return useQuery<FuturesPosition[] | []>(
+	return useQuery<any>(
```

Comment:
If we're not returning anything I think we should change to undefined rather than any.

Reply:
Need to return `void`, but good call